addons/point_of_sale/models/pos_config.py

```diff
@@ -213,6 +213,7 @@ def _compute_current_session(self):
         """If there is an open session, store it to current_session_id / current_session_State.
         """
         for pos_config in self:
+            pos_config.session_ids.fetch(["state"])
```

Comment:
```suggestion
        self.session_ids.fetch(["state"])
        for pos_config in self:
```

will this also work?

Reply:
@caburj Thank you for the review! The suggested changes work.